### DISCUSSION TITLE
Fix that you able to increase oxygen tank's effective time of use

### DISCUSSION THREAD

**Discussion by Abkim27** [FakeFishGames/Barotrauma#12350] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

You are able to do this

https://github.com/Regalis11/Barotrauma/assets/100293037/288c1744-01c3-4e02-bb7b-bf82f44a35cc

to increase overall time of use of one oxygen tank. My fix is to remove some of oxygen tank's condition when taking it out of suit.

### Reproduction steps

1.Get into water in diving suit.
2.Take out oxygen tank.
3.Wait.
4.Before you pass out put oxygen tank back.
5.Now you "increased" oxygen tank's capacity!

### Bug prevalence

Happens every time I play

### Version

Unstable (v1.1.3.0)

### Which operating system did you encounter this bug on?

Linux

**Comment by Abkim27** [comment]
This thing is clearly not intended and so i think it this is a issue, not feature request

**Comment by zomgtehderpy** [comment]
doubling (or more) the recovery time from "o2 low" compared to gaining it could also solve this "exploit". looking at current source, it seems the other way around: "o2 low" gain is two times slower than recovery.

**Reply by jigardi** [reply]
Not sure how I feel about this, but I think it deserves some more consideration at least.
